src/core/bitop.d

```diff
@@ -414,3 +414,75 @@ unittest
     foreach(i; 0 .. 32)
         assert(bitswap(1 << i) == 1 << 32 - i - 1);
 }
+
+/**
+ * Reverses the order of bits in a 64-bit integer.
+ */
+ulong bitswap ( ulong x ) pure @trusted
+{
+    version (D_InlineAsm_X86_64)
+    {
+        asm pure nothrow @nogc { naked; }
+
+        version (Win64)
+            asm pure nothrow @nogc { mov RAX, RCX; }
+        else
+            asm pure nothrow @nogc { mov RAX, RDI; }
+
+        asm pure nothrow @nogc
```

Comment:
it is same for existing `uint` overload https://github.com/D-Programming-Language/druntime/pull/1374/files#diff-6d81a130202753008e7b435a4ca71ebcL354